src/libs/EmojiTrie.js

```diff
@@ -0,0 +1,31 @@
+import emojis from '../../assets/emojis';
+import Trie from './Trie';
```

Comment:
NAB but I think it might be a good idea to use the `src/libs/actions/Timing` lib to keep track of how long this is taking. Would probably be a very easy change to start a timer before creating the `new Trie()` and stop the timer after inserting all the emoji into it.

Reply:
Done.